configure.ac

```diff
@@ -1037,6 +1037,21 @@ if echo $host_os | grep '^darwin' >/dev/null ; then
   if test `uname -r | sed 's/\.//g'` -lt 800 ; then
     AC_DEFINE_UNQUOTED(DARWIN_10_3, [1],[Define to 1 if compiling under OS X 10.3 or earlier])
   fi
+  AS_IF([test "${enable_client}" = "yes" || test "${enable_manager}" = "yes"],
```

Comment:
It is supported. It's still, let's say. not official, but there were put a lot of effort to add ability to build BOINC with Autotools. Also it's built now on Travis CI exactly using Autotools

Reply:
Unfortunately I don't have a Mac to test this. Its unlikely to work given how #2780 made the changes to build the missing components to "supposedly" work under Linux which wasn't meant to use there. When I make this commit, I am basing around https://github.com/BOINC/boinc/pull/2780#issuecomment-433253013